### DISCUSSION TITLE
Created field change

### DISCUSSION THREAD

**Discussion by sabikrahat** [pocketbase/pocketbase#3248] [Q&A] [answered]
Is there any way to change the `created` field value from flutter application? I'm passing the `created` 

```
await pb.collection(vehicles).create(
      body: {
        ......
        'creator': pb.authStore.model!.id,
        'created': DateTime.now().addDays(5).toUtc().toIso8601String(),
      },
    )
```

 I've pass the value of created like this for testing. I'll take this value from the user. But in db it is storing the `DateTime.now()` by default.

**Comment by ganigeorgiev** [comment, marked as answer]
No, but you can add your own date field (eg. `submitted`/`published`) instead.

At the moment the system `created`/`updated` fields are automanaged and can't be modified with the web APIs (_this may change in the future_).

Alternatively you can attach to the record before create hook with Go or JS and manually set the `created` field with the submitted body data. Here is an example with JS:

```js
// pb_hooks/main.pb.js
onRecordBeforeCreateRequest((e) => {
    const info = $apis.requestInfo(e.httpContext);

    e.record.set('created', info.data.created);
})
```
